Makefile

```diff
@@ -0,0 +1,34 @@
+.DEFAULT_GOAL := compile
```

Comment:
-1 on `make`.

The project setup is pretty simple and there is no need for the `make` abstraction.